vm/src/frame.rs

```diff
@@ -1,1542 +1,1542 @@
-use std::fmt;
```

Comment:
You keep reformatting this file to use crlf instead of lf line endings; would you be able to figure out what's causing that? It's hard to look at the diffs when it shows the diff as the whole file.

Reply:
🤔 that is weird. Sure. I am using VSCode with Rust extension. Will force to lf and push again.

Reply:
CRLF fixed on ee54f3b